src/pages/example.tsx

```diff
@@ -0,0 +1,16 @@
+// src/pages/example
+export default function Example({ someProps }) {
+  return (
+    <StandardPage
+      backgroundColor="pale-blue"
+      meta={{
+        // seo shenanigans
+        title: someProps.title,
+      }}
+    >
+      <ExampleScene {...props} />
+    </StandardPage>
+  );
+}
+
+// (also getServerSideProps, getStaticProps, etc.)
```

Comment:
We added this new page as an example towards the end as a suggestion for a nice clean Next.js pages/props structure:

* `src/pages` contains only these very basic components (no logic!) and their associated data fetchers, such as `getServerSideProps`
* `src/components/scenes` holds the equivalent "scenes" components (e.g. `src/components/Example for src/pages/example`
* GraphQL queries might go into `src/pages` if you can configure Next.js to not serve them as pages
    * Auto-generated types are generated into separate files you can import in the scenes